src/d8/default.settings.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * @file
+ * Drupal site-specific configuration file.
+ */
+
+/**
+ * Include secret configuration.
+ *
+ * Contains database settings and other sensitive environment specific
+ * information that shouldn't be in version control.
+ */
+if (file_exists(DRUPAL_ROOT . '/sites/default/secret.settings.php')) {
+  include DRUPAL_ROOT . '/sites/default/secret.settings.php';
+}
+
+/**
+ * Master module configuration.
+ * @see https://www.drupal.org/project/master
+ */
+$conf['install_profile'] = 'standard';
+$conf['master_version'] = 2;
+$conf['master_modules'] = array(
```

Comment:
We need to go through this list for D8 and make sure that we a) disable the right core modules and b) enable the right custom modules. For instance, we don't download Fences for D8 (yay twig!)